src/components/Tabs/index.stories.js

```diff
@@ -11,6 +11,7 @@ stories.add('Playground', () => (
   <>
     <Heading>Tabs</Heading>
     <Tabs
+      defaultIndex={1}
```

Comment:
Shouldn't it be `0`?

Reply:
Well, it's to showcase that you can select any index. So it can be anything.

Reply:
My bad I thought it was done on the component side 🤦 
In that case, could you create a new section to showcase this?

```
<Heading>Tabs</Heading>
...
<Heading level={2}>With default index</Heading>
    <Tabs defaultIndex={1} ... />

```